web/src/components/vis/MetaboliteTable.vue

```diff
@@ -34,17 +37,29 @@ export default {
 </script>
 
 <template lang="pug">
-v-data-table.elevation-1(:headers="headers", :items="items", disable-initial-sort,
-    item-key="Metabolite")
+v-data-table.elevation-1.main(:headers="headers", :items="items", disable-initial-sort,
+    item-key="Metabolite", :pagination.sync="pagination")
   template(v-slot:items="props")
-    td {{ props.item.Metabolite }}
-    td.text-xs-right(v-for="c in headers.slice(1)",
+    td.cell {{ props.item.Metabolite }}
+    td.cell.text-xs-right(v-for="c in headers.slice(1)",
         :class="isInteresting(props.item[c.value]) ? 'highlight' : ''")
       | {{ format(props.item[c.value]) }}
 </template>
 
-<style lang="scss" scoped>
+<style scoped>
 .highlight {
   background-color: #ffadad;
 }
+
+.main {
+  height: 0;
+}
+
+.main >>> tr {
+  height: 25px;
+}
+.main >>> td.cell {
```

Comment:
I'm a big fan of this.  I usually end up using vuetify's internal class names and it bites me when they change.  I like that this is an alternative option.